cmd/cluster/create.go

```diff
@@ -304,3 +314,25 @@ func CreateCluster(ctx context.Context, opts Options) error {
 
 	return nil
 }
+
+func generateSSHKeys() ([]byte, []byte, error) {
```

Comment:
I am not a big fan of doing this implicitly and by default, I would at least want an opt-in. Also nit, but ECDSA keys are much cheaper to generate computationally